module/dkg/broker.go

```diff
@@ -204,18 +209,38 @@ func (b *Broker) Broadcast(data []byte) {
 }
 
 // SubmitResult publishes the result of the DKG protocol to the smart contract.
+// This function should be passed the beacon keys (group key, participant keys) resulting from the DKG process.
+// If the DKG process failed, no beacon keys will exist. In that case, we pass in nil here for both arguments.
+//
+// If non-nil arguments are provided, we submit a non-empty ResultSubmission to the DKG smart contract,
+// indicating that we completed the DKG successfully and essentially "voting for" our result.
+// If nil arguments are provided, we submit an empty ResultSubmission to the DKG smart contract,
+// indicating that we completed the DKG unsuccessfully.
 func (b *Broker) SubmitResult(groupKey crypto.PublicKey, pubKeys []crypto.PublicKey) error {
```

Comment:
here, we are actually only submitting the results. So I agree with the method name. Just wanted to mention that this method should therefore _not_ be renamed. 

Thanks for the detailed documentation 🙇